mono/mini/mini-amd64.h

```diff
@@ -220,28 +220,31 @@ typedef struct MonoCompileArch {
 
 #ifdef TARGET_WIN32
 
-static AMD64_Reg_No param_regs [] = { AMD64_RCX, AMD64_RDX, AMD64_R8, AMD64_R9 };
+static G_GNUC_UNUSED AMD64_Reg_No param_regs [] = { AMD64_RCX, AMD64_RDX, AMD64_R8, AMD64_R9 };
```

Comment:
Then declaring them becomes rather cumbersome since you have to explicitly specify the array size.